script/tool/test/util.dart

```diff
@@ -105,8 +105,11 @@ Directory createFakePackage(
   }
 
   final FileSystem fileSystem = packageDirectory.fileSystem;
-  for (final List<String> file in extraFiles) {
-    final List<String> newFilePath = <String>[packageDirectory.path, ...file];
+  for (final String file in extraFiles) {
+    final List<String> newFilePath = <String>[
+      packageDirectory.path,
+      ...p.split(file)
```

Comment:
Is this going to work when running the tests on windows?  Since the tests will always be written with posix paths we could just split on '/'.

Reply:
Good catch, I misremembered the default behavior of `path`; I'll fix it to initialize with an explicit style.